models.py

```diff
@@ -1,9 +1,8 @@
-from app import db
 from sqlalchemy import Column, Integer, String, ForeignKey, DateTime
-from sqlalchemy.orm import declarative_base, validates
+from sqlalchemy.orm import validates
+
+from app import db
 
-# declarative base class
-Base = declarative_base()
```

Comment:
Base is unused, so I removed it. It looks like Flask-SQLAlchemy basically sets this up for you:
https://stackoverflow.com/questions/22698478/what-is-the-difference-between-the-declarative-base-and-db-model